package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "start": "./bin.js example.js --open -p 8080 -A 0.0.0.0",
+    "build": "NODE_ENV=production ./bin.js build example.js --optimize",
```

Comment:
Hmmm, yeah actually I agree a little bit here; I think it might make sense to always run `envify` so that `process.env.NODE_ENV` is always respected. That way people can always run:
```js
if (process.env.NODE_ENV === 'production') // do foo
else // do bar
```